site/docs/reference/Connectors/capture-connectors/MySQL.md

```diff
@@ -84,7 +84,7 @@ You're able to apply the connector directly to the primary instance if you'd lik
 
 1. Allow connections to the database from the Estuary Flow IP address.
 
-   1. [Modify the database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Modifying.html), setting **Public accessibility** to **Yes**.
+   1. [Modify the database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Modifying.html), setting **Public accessibility** to **Yes**.  See the directions below to use a SSH Tunnel instead of enabling public access.
```

Comment:
Yes, any production use case generally involves using SSH and people tend to test with a publicly accessible IP.  I've been told by several people that they were not interested in the product because it "required opening IPs to public" so wanted to make that clear.  Would love it if you could help out post merge!